clean_bucket/adt/1993010100.yaml

```diff
@@ -0,0 +1,111 @@
+aws_s3:
+     adt.cryosat2:
+
+          # Define the path attributes.
+          local_path: !ENV ${WORKrnr}/${EXPTrnr}/${CYCLE}/intercom/inputs/obs/ocean/cryosat2.adt.iodav2.nc
+          offset_seconds: 21600
+
+          # Define the AWS s3 attributes.
+          bucket: noaa-reanalyses-pds
+          object_path: observations/reanalysis/adt/cryosat2/%Y/%m/iodav2/cryosat2.adt.%Y%m%d.T120000Z.iodav2.nc
```

Comment:
i am on a fence about it. But would it make it more clear if we have two sections:
```
source:
  bucket: 
  object_path:
  profile_name
  ignore_missing:

```
and 
```
destination:
   local_path:
   offset_seconds:
```